bin/kano-ui-autostart

```diff
@@ -115,12 +117,29 @@ kill %?xrefresh
 
 # TODO: move me to a systemd unit service
 if [ "$FIRST_BOOT" -eq 1 ]; then
```

Comment:
I don't think the audio stuff needs to be conditioned on FIRST_BOOT: this script is only run once per login anyway, right? And we don't want to limit this to the actual first boot after account setup.